OrbitLinuxTracing/PerfEvent.h

```diff
@@ -185,6 +185,36 @@ struct dynamically_sized_perf_event_stack_sample {
       : stack{dyn_size} {}
 };
 
+class CallchainSamplePerfEvent : public PerfEvent {
+ public:
+  perf_event_callchain_sample ring_buffer_record;
+  std::vector<uint64_t> ips;
+  explicit CallchainSamplePerfEvent(uint64_t nr) : ips(nr) {}
```

Comment:
Yes, this is what I had in mind. Now you can use more descriptive names if you want (I know what `nr` is, but it's a bit of an unexpected name).